### PR TITLE
fetch the list of changed files

### PR DESCRIPTION
**Pull request by ganganandikonda**

### CODE REVIEW

File: TeamcityRestApi/JsonTypes.fs [hunk 1]
```diff
@@ -1313,16 +1313,54 @@ type Changes = JsonProvider<"""
             "username": "tran tu",
             "date": "20180820T074849+0300",
             "href": "/app/rest/changes/id:2776374",
-            "webUrl": "http://teamcity/viewModification.html?modId=2776374&personal=false"
+            "webUrl": "http://teamcity/viewModification.html?modId=2776374&personal=false",
+           "files":
```
Comment (jmecosta) [inline TeamcityRestApi/JsonTypes.fs:1317]: indentation incorrect

File: TeamcityRestApi/TeamcityConnector.fs [hunk 2]
```diff
@@ -977,6 +977,11 @@ type TeamcityConnector(httpconnector : IHttpTeamcityConnector) =
                             changedata.Version <- change.Version
                             changedata.WebUrl <- change.WebUrl
                             
+                            if change.Files.Count <> 0 then
+                                let mutable changedfile = ""
+                                for file in change.Files.File do
+                                    changedfile <- file.File
+                                    changedata.ListofChangedFiles.Add(changedfile)
```
Comment (jmecosta) [inline TeamcityRestApi/TeamcityConnector.fs:984]: space after this block

File: TeamcityRestTypes/Change.cs [hunk 3]
```diff
@@ -6,6 +6,9 @@
 //   The tc build.
 // </summary>
 // --------------------------------------------------------------------------------------------------------------------
+using System;
```
Comment (jmecosta) [inline TeamcityRestTypes/Change.cs:9]: usings inside namespace

File: TeamcityRestTypes/Change.cs [hunk 4]
```diff
@@ -69,5 +72,20 @@ public class Change
...
+        /// <value>
+        /// The changed files.
+        /// </value>
+        public List<string> ListofChangedFiles { get; set; }
+
+        public Change()
+        {
+            this.ListofChangedFiles = new List<string>();
+        }
+
```
Comment (jmecosta) [inline TeamcityRestTypes/Change.cs:88]: extra dead space

**Review by jmecosta — CHANGES_REQUESTED** [review]

File: TeamcityRestApi/JsonTypes.fs (hunk 1 above)
Comment (ganganandikonda) [reply TeamcityRestApi/JsonTypes.fs]: fixed

File: TeamcityRestApi/TeamcityConnector.fs (hunk 2 above)
Comment (ganganandikonda) [reply TeamcityRestApi/TeamcityConnector.fs]: fixed

File: TeamcityRestTypes/Change.cs (hunk 3 above)
Comment (ganganandikonda) [reply TeamcityRestTypes/Change.cs]: fixed

File: TeamcityRestTypes/Change.cs (hunk 4 above)
Comment (ganganandikonda) [reply TeamcityRestTypes/Change.cs]: fixed

**Review by jmecosta — APPROVED** [review]

**Pull request merged**